app/api/stripe/paymentIntent/route.ts

```diff
@@ -0,0 +1,26 @@
+import { NextRequest, NextResponse } from "next/server";
+import { stripe } from "@/utils/stripe";
+
+interface PaymentIntentParams {
+  amount: number;
+}
+export async function POST(req: NextRequest) {
+  const { amount }: PaymentIntentParams = await req.json();
+  try {
+    const paymentIntent = await stripe.paymentIntents.create({
+      amount: amount,
+      currency: "usd",
+      // Verify your integration in this guide by including this parameter
```

Comment:
comment (appears) pasted from llm/tutorial not needed

Reply:
:) oups https://docs.stripe.com/stripe-js/react .. got it from their docs